src/ol/interaction/Extent.js

```diff
@@ -218,6 +229,10 @@ class Extent extends PointerInteraction {
    * @private
    */
   handlePointerMove_(mapBrowserEvent) {
+    if (!this.condition_(mapBrowserEvent)) {
+      this.removePointFeature_();
+      return;
```

Comment:
`return false` here to indicate that the event has not been consumed by this interaction.